### ISSUE TITLE
How does one install this app in their Android phone?

### ISSUE THREAD

**Issue by answerquest** [Avinashkumarsharma/noteapp#1]
There's no .apk file listed...

**Comment by Avinashkumarsharma** [comment]
This was made to run in browser. This can be easily ported to android.

**Comment by answerquest** [comment]
Thanks for replying! So I can copy all the files to a folder on the phone and run index.html?

**Comment by Avinashkumarsharma** [comment]
No, I dont think that will work. Upload this to a server and open or port this using Webview in android
